sparse/etc/pulse/arm_droid_default.pa

```diff
@@ -23,7 +23,12 @@
 
 .fail
 
-load-module module-droid-keepalive
+# Transitional check as keepalive module is moved away from droid modules.
+.ifexists module-keepalive
```

Comment:
Should this be ".ifexists module-keepalive.so", all similar conditions below always have .so.

Reply:
Actually .so can be omitted, but it's more explicit to use .so, I'll update the PR.